.travis.yml

```diff
@@ -0,0 +1,16 @@
+branches: {only: [master, develop]}
+dist: xenial
+sudo: false
+language: python
+python:
+  - 3.7
```

Comment:
I've been using python 3.8 for this project so far since that's the newest and I don't see a reason to use 3.7.